jax/experimental/jax2tf/jax2tf.py

```diff
@@ -218,6 +218,30 @@ def _interpret_fun(fun: lu.WrappedFun,
     del main
   return out_vals
 
+def _convert_jax_impl(jax_impl: Callable, multiple_results=True) -> Callable:
+  """Convert the JAX implementation of a primitive.
+
+  Args:
+    jax_impl: typically the impl-rule for a primitive, with signature
+      `(*args: JaxVal, **kwargs) -> Sequence[JaxVal]`. This function implements
+      a primitive in terms of other primitives.
+    multiple_results: whether `jax_impl` returns a sequence of results.
+
+  Returns:
+     a function with signature `(*args: TfValOrUnit, **kwargs) -> Sequence[TfValOrUnit]`.
+  """
+  def wrapped(*tf_args: TfValOrUnit, **kwargs) -> Sequence[TfValOrUnit]:
```

Comment:
Is it OK to make this return `Union[TfValOrUnit, Sequence[TfValOrUnit]]` to satisfy mypy, as when `multiple_results == False` then the return type is not a Sequence.

Reply:
I'll try that, but in the past we got strange failures from pytype when we started to use more `Union`.